etc/config/java.amazon.properties

```diff
@@ -15,19 +15,40 @@ baseName=jdk
 licenseName=The GNU General Public License (GPL) Version 2
 licenseLink=https://github.com/openjdk/jdk/blob/master/LICENSE
 
+group.java8.compilers=java8u402b06
+group.java8.groupName=OpenJDK 8
+group.java8.objdumper=/opt/compiler-explorer/jdk-8u402b06/bin/javap
+compiler.java8u402b06.exe=/opt/compiler-explorer/jdk-8u402b06/bin/javac
+compiler.java8u402b06.runtime=/opt/compiler-explorer/jdk-8u402b06/bin/java
+compiler.java8u402b06.semver=8.402.6
+
+group.java9.compilers=java904
+group.java9.groupName=OpenJDK 9
+group.java9.objdumper=/opt/compiler-explorer/jdk-9.0.4/bin/javap
+compiler.java904.exe=/opt/compiler-explorer/jdk-9.0.4/bin/javac
+compiler.java904.runtime=/opt/compiler-explorer/jdk-9.0.4/bin/java
+compiler.java904.semver=9.0.4
+
+group.java10.compilers=java1002
+group.java10.groupName=OpenJDK 10
+group.java10.objdumper=/opt/compiler-explorer/jdk-10.0.2/bin/javap
+compiler.java1002.exe=/opt/compiler-explorer/jdk-10.0.2/bin/javac
+compiler.java1002.runtime=/opt/compiler-explorer/jdk-10.0.2/bin/java
+compiler.java1002.semver=10.0.2
+
 group.java11.compilers=java1102
 group.java11.groupName=OpenJDK 11
 group.java11.objdumper=/opt/compiler-explorer/jdk-11.0.2/bin/javap
 compiler.java1102.exe=/opt/compiler-explorer/jdk-11.0.2/bin/javac
 compiler.java1102.runtime=/opt/compiler-explorer/jdk-11.0.2/bin/java
 compiler.java1102.semver=11.0.2
 
-group.java12.compilers=java1201
+group.java12.compilers=java1202
```

Comment:
I believe you should add 1202 in additon to 1201 here (and consequently for the other versions too). Any shortlinks using 1201 will break if we remove the old compiler ids.